packages/flutter/lib/src/widgets/form.dart

```diff
@@ -367,7 +367,9 @@ class FormState extends State<Form> {
       if (!validateOnFocusChange || !field._focusNode.hasFocus) {
         final bool isFieldValid = field.validate();
         hasError = !isFieldValid || hasError;
-        errorMessage += field.errorText ?? '';
+        if (errorMessage.isEmpty) {
```

Comment:
You can add a comment here for this change.

Reply:
done!

Reply:
nit:  The comment `// Ensure that only the first error message gets announced to the user.` should be one line before 
`if (errorMessage.isEmpty) {`

Reply:
This is not updated.